hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -58,15 +65,39 @@ protected boolean requiresRuntime(OperationContext context) {
     @Override
     protected void removeServices(OperationContext context, ServiceName parentService, ModelNode parentModel)
```

Comment:
I think this is good as is (assuming it all works :)) No need to put this in Util yet. Don't think we'll have a need for it - but if we do, we can refactor then.

I assume this works on EAP 6.4 as well as 7.0 and 7.1?

Let's see if we can get an itest in for this before merging. 

@pilhuhn  asked for an agent release today so we'll need to get something merged today to fix this problem before we release.